### ISSUE TITLE
"verify_amqp_uri" in the module "nameko.amqp.utils" had been removed?

### ISSUE THREAD

**Issue by XiaoYang1127** [nameko/nameko#761]
### environment
- linux debian9+
- python3.7.5

### library
- nameko==2.14.1
- nameko-proxy==0.0.5
- Flask==2.1.2

##### main.py
```python
from eventlet import monkey_patch
from flask import Flask
from nameko_proxy.wrappers.flask import FlaskNamekoProxy

rpc = FlaskNamekoProxy()


def make_app(conf_path: str = 'settings.py') -> Flask:
    app = Flask(__name__)
    x = app.config
    x.from_pyfile(conf_path)
    print(x.items())
    return app


def start_app():
    app = make_app()
    rpc.init_app(app)

    monkey_patch()
    app.run()


if __name__ == '__main__':
    start_app()
```

##### settings.py

```python
"""_summary_
- Wrapper get from flask config all parameters started from prefix "NAMEKO_" and pass it to ClusterRpcProxy
- all parameters must be upper
"""

NAMEKO_AMQP_URI = "amqp://root:root@localhost:5672/"
NAMEKO_RPC_TIMEOUT = 1
NAMEKO_PREFETCH_COUNT = 10
```

### errors

Traceback (most recent call last):
  File "test_flask.py", line 3, in <module>
    from nameko_proxy.wrappers.flask import FlaskNamekoProxy
  File "/home/web/prj/py_ms3/venv/lib/python3.7/site-packages/nameko_proxy/__init__.py", line 1, in <module>
    from nameko_proxy.proxy import *
  File "/home/web/prj/py_ms3/venv/lib/python3.7/site-packages/nameko_proxy/proxy.py", line 7, in <module>
    from nameko_proxy.reply_listener import StandaloneReplyListener
  File "/home/web/prj/py_ms3/venv/lib/python3.7/site-packages/nameko_proxy/reply_listener.py", line 3, in <module>
    from nameko_proxy.queue_consumer import QueueConsumer
  File "/home/web/prj/py_ms3/venv/lib/python3.7/site-packages/nameko_proxy/queue_consumer.py", line 8, in <module>
    from nameko.amqp import verify_amqp_uri
ImportError: cannot import name 'verify_amqp_uri' from 'nameko.amqp' (/home/web/prj/py_ms3/venv/lib/python3.7/site-packages/nameko/amqp/__init__.py)

**Comment by XiaoYang1127** [comment]
it had beed solved  when I downgraded nameko from 2.14.1 to 2.11.0

**Comment by mattbennett** [comment]
It was removed here https://github.com/nameko/nameko/commit/3ba7418232adbd2a8a87dfc1f5a59a985efad90b
